packages/admin/dashboard/src/components/table/table-cells/common/date-cell/date-cell.tsx

```diff
@@ -13,10 +13,9 @@ export const DateCell = ({ date }: DateCellProps) => {
   }
 
   const value = new Date(date)
-  value.setMinutes(value.getMinutes() - value.getTimezoneOffset())
 
   const hour12 = Intl.DateTimeFormat().resolvedOptions().hour12
-  const timestampFormat = hour12 ? "dd MMM yyyy hh:MM a" : "dd MMM yyyy HH:MM"
+  const timestampFormat = hour12 ? "dd MMM yyyy hh:mm a" : "dd MMM yyyy HH:mm"
```

Comment:
suggestion: Use the getFullDate function from `useDate` instead so we ensure that we use the same format as we do for all other dates